extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java

```diff
@@ -75,6 +75,10 @@ private static File extractFile (String sourcePath, File outFile) {
 			OutputStream out = null;
 			InputStream in = null;
 
+			if (outFile.exists()) {
+				return outFile;
+			}
```

Comment:
There's not much to say here; this is a reasonable check to have. Because different, incompatible versions of native libraries should pretty much always have different checksums, this should never encounter any problem with two different versions getting loaded by different applications. Approved!